types/office-js-preview/index.d.ts

```diff
@@ -11946,6 +11948,14 @@ declare namespace Office {
          *
          * **Important**: In Outlook on the web, `setSignatureAsync` only works on messages.
          *
+         * **Important**: If your add-in implements the 
+         * {@link https://docs.microsoft.com/office/dev/add-ins/outlook/autolaunch | event-based activation feature using `LaunchEvent` in the manifest},
+         * and calls `setSignatureAsync` in the event handler, the following behaviors apply.
+         *
+         * - If the user is editing a meeting, the `setSignatureAsync` call returns an error.
```

Comment:
I believe Daniele fixed the behavior to be that autorun does not run at all on calendar edits and drafts. So the behavior should not be specific to setSignature anymore, but a change to the AutoRun feature (does not run when it is launched while editing a calendar items or opening a draft).

Reply:
@danielebercovici Can you take a look and confirm current/expected behavior? Thanks.

Reply:
Emily is right, autorun add-ins will not run on anything that isn't a new compose (draft/calendar edits).

Reply:
I removed the statement about editing a meeting. I'll update the autorun article to note this behavior.